### PR TITLE
licenses: update copyright year

### PR DESCRIPTION
**Pull request by alyssais**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [x] Have you successfully run `brew tests` with your changes locally?

-----

Happy 2017 😊

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Small note, this does not need to continuously be updated, we can just remove the ending year. To be valid, the notice requires only the first year.

**Comment by alyssais** [comment]
That sounds a lot more sensible.

**Comment by MikeMcQuaid** [comment]
Agreed. The Homebrew Cask license can also now be removed given they are a single codebase.

File: LICENSE.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 BSD 2-Clause License
 
-Copyright (c) 2009-2016, Homebrew contributors
+Copyright (c) 2009, Homebrew contributors
```
Comment (MikeMcQuaid) [inline LICENSE.txt]: Pedantry: `2009-Present` or similar reads better

**Comment by alyssais** [comment]
Oh, I updated this. (Force pushed, so you wouldn't have got a notification.)

**Comment by MikeMcQuaid** [comment]
hurrah, thanks @alyssais.

**Pull request merged**